### ISSUE TITLE
PlantUml diagram in AsciiDoc-files do not get rendered using docker image v 1.1.8

### ISSUE THREAD

**Issue by mcbeelen** [avisi-cloud/structurizr-site-generatr#453]
The page about[ AsciiDoc features](https://github.com/avisi-cloud/structurizr-site-generatr/blob/main/docs/example/workspace-docs/03-asciidoc-features.adoc) in this repo indicate that is should be possible to include plantuml in the AsciiDoc and get the rendered diagrams as part of the static site.

This does not appear to work with the docker image v1.1.8

### Example

Given an index.adoc with the following content:

Example file
```
= Index

[plantuml]
----
@startuml

Alice -> Bob : Hello

@enduml
----
```

Generating the site with
`docker run -it --rm -p 8080:8080 -v %CD%:/var/model ghcr.io/avisi-cloud/structurizr-site-generatr:1.1.8 generate-site -w workspace.dsl`

### Result 
![plantuml-not-rendered](https://github.com/avisi-cloud/structurizr-site-generatr/assets/1905153/8cb69bd4-0e65-4ef7-8da3-8fee9ab52fa1)

## Additional note

I did a git checkout of the repo of this project and ran the command in the `docs/example`-directory and then the examples from the 03-asciidoc-features.adoc are not rendered as well.

**Comment by jp7677** [comment]
Rendering PlantUml diagrams in AsciiDoc/Markdown files was merged after 1.1.8, so correct that it isn't working in the release version. That said, if you are running `main`, it should be there.

I just did a test myself with `main` and the example in the repository, it worked as expected.

![out](https://github.com/avisi-cloud/structurizr-site-generatr/assets/17219293/12a06f40-2afb-49bd-b17a-bdb4dc4a7721)
